cpp/src/structure/detail/structure_utils.cuh

```diff
@@ -83,6 +83,38 @@ struct rebase_offset_t {
   __device__ edge_t operator()(edge_t offset) const { return offset - base_offset; }
 };
 
+template <typename vertex_t, typename edge_t>
+rmm::device_uvector<vertex_t> expand_sparse_offsets(raft::device_span<edge_t const> offsets,
+                                                    vertex_t base_vertex_id,
+                                                    rmm::cuda_stream_view stream_view)
+{
+  edge_t num_edges{0};
+
+  if (offsets.size() > 0) {
```

Comment:
So, offsets.size() should be at least 0 + 1, so I guess we'd better assume offsets.size() > 0? (add assert(offsetes.size() > 0?)

Reply:
There was - during my debugging - a case in MG where on some of the partitions the input was empty, I added this check to make sure that didn't result in a crash.  I will change it to an assert.

Reply:
Even if the input is empty (i.e. # subgraphs is 0), the offsets.size() should be 0 + 1 = 1 and > 0, right? In this case offsets should be [0] if I am not mistaken.

Reply:
If it is specified correctly :-) . I believe it was a bug in my test code which results in the error.